recipes-kernel/dkms/dkms.bb

```diff
@@ -18,15 +18,26 @@ SRC_URI = "\
 SRCREV = "4d466bf727347408307aa28ab4f090488360b592"
 
 
-inherit autotools-brokensep
+# dkms does not ship a configure script and is not autotools-based.
+# Running standard oe_runconf/autotools tasks results in errors:
+#   ./configure: No such file
+# Therefore skip both tasks and call `make install` manually.
+do_configure[noexec] = "1"
+do_compile[noexec] = "1"
+
+
+do_install() {
+	oe_runmake install DESTDIR=${D} LIBDIR=${libdir} 
+}
+
 
 # We don't need the dist/ tarball.
 EXTRA_OEMAKE += " -o tarball"
 
 INSANE_SKIP:${PN} += "dev-deps"
 
 
-FILES:${PN} += " ${datadir}/bash-completion/* ${datadir}/zsh/*"
+FILES:${PN} += " ${datadir}/bash-completion/* ${datadir}/zsh/* ${libdir}/dkms_autoinstaller ${libdir}/common.postinst"
```

Comment:
Did you compare packages from this recipe on scarthgap vs dist-next to ensure everything remains same?